app/src/main/java/org/stepic/droid/ui/activities/SocialAuthActivity.kt

```diff
@@ -0,0 +1,70 @@
+package org.stepic.droid.ui.activities
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import org.stepic.droid.R
+import org.stepic.droid.util.AppConstants
+
+
+class SocialAuthActivity  : AppCompatActivity() {
```

Comment:
add `loader` to `WebView`, not `WebView` is white about 3-5 seconds. It might be confusing to the user.